### PR TITLE
parsing as <li> elements

### PR DESCRIPTION
**Pull request by Pomax**
This works with the TeachAPI PR for getting marked up badge information, https://github.com/mozilla/teach-api/pull/75, and parses the badge criteria and evidence requirements based on getting lists of `<li>` items.

### CODE REVIEW

File: pages/badges/badge-single.jsx [hunk 1]
```diff
@@ -133,9 +133,12 @@ var BadgePage = React.createClass({
 
     var evidence = [];
     var criteria = [];
-
-    // extract evidence as itemized list based on newlines (\n with optional \r) if we can
     var splitOnItems = /\n\r?/g;
+    var parser = document.createDocumentFragment();
+    var pbody = document.createElement("body");
+
```
Comment (gideonthomas) [inline pages/badges/badge-single.jsx]: extra line

**Comment by Pomax** [comment]
Virtually all other PRs have landed, so we should be able to test this on the master codebase now.

**Comment by gideonthomas** [comment]
testing now

**Comment by gideonthomas** [comment]
this works well! so r+

**Pull request merged**